<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа №4</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Введите число элементов : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_n"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_gen">
        <property name="text">
         <string>Сгенерировать массивы</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>a = </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="list_a"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>b = </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="list_b"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>form_04.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_go">
      <property name="text">
       <string>Сосчитать</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>c = </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="list_c"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>line_n</tabstop>
  <tabstop>btn_gen</tabstop>
  <tabstop>list_b</tabstop>
  <tabstop>list_a</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
